addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -280,6 +300,12 @@ function factory(dependencies) {
         hasFollowers: attr({
             default: true,
         }),
+        /**
+         * Determine whether to display a message list for this chatter.
+         */
+        hasMessageList: attr({
+            default: true,
+        }),
```

Comment:
A bit confusing to have `hasThread(Viewer)` and `hasMessageList`, which work quite differently (one is a decisive customization, the other is purely computed value)... But honestly not a big deal.

Reply:
`hasThreadViewer` is decisive too, it's just that it's computed instead of being set manually. (note that setting it manually will work to create the thread view, at least until one of the dependency changes)

But there are actually other cases where indeed some boolean are informative rather than decisive. I usually differentiate them with a different docstring ("states..." for informative and "determines..." for decisive, but that guideline was not applied previously)